conda/exceptions.py

```diff
@@ -552,6 +552,16 @@ def __init__(self, message):
         super(RemoveError, self).__init__(msg)
 
 
+class DisallowedError(CondaError):
```

Comment:
DisallowedPackagesError would make it more explicit? or do we plan to use it for disallowing other things?

Reply:
ok, making that change now